### PR TITLE
Run3-gex10 Recreate Run3 scenarios with the new xml files

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Recreate Run3 scenarios with the xml files where the material files are reaffarnged

#### PR validation:

Tested with the workflow 11634.0 in runTheMatrix and then reading the scenario from XML files

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by cvuosalo** [comment]
urgent

File: Configuration/Geometry/README.md [hunk 1]
```diff
@@ -15,16 +17,17 @@ Muon system:
...
-* O1: 2021 baseline
-* O2: as O1, but with zero material
+* O3: 2021 baseline
+* O4: as O3, but with zero material
 * F1: 2021 baseline
 
 Several detector combinations have been generated:
-* 2021 = T1+C1+M1+P1+O1+F1
-* 2021ZeroMaterial = T2+C1+M1+P1+O2+F1
+* 2021 = T3+C1+M1+P2+O3+F1
```
[the remark this answers is not in the document]
Comment (cvuosalo) [reply Configuration/Geometry/README.md:29]: @bsunanda Could you try to respond to Kevin's questions?

File: Configuration/Geometry/README.md [hunk 2]
```diff
@@ -15,16 +17,17 @@ Muon system:
...
 
 PPS:
-* P1: 2021 baseline
+* P1: 2021 baseline (old)
+* P2: 2021 baseline (after using its own material files for pixel)
 
 The script also handles the common and forward elements of the geometry:
-* O1: 2021 baseline
-* O2: as O1, but with zero material
+* O3: 2021 baseline
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Configuration/Geometry/README.md:24]: I changed that

File: Configuration/Geometry/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bsunanda) [reply Configuration/Geometry/README.md:29]: I hope it is allowed. We are not versioning 2021 geometries. The basic design is not changed. Only the materials are rearranged for future usage (studies of tracker material budget by changing densities of tracker specific materials)

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
@bsunanda thanks, a few followups:
* add `,deprecatedSubdets=deprecatedSubdets` to the end of this line: https://github.com/cms-sw/cmssw/blob/ff36edc9f52a8f0b55f365fe96dea251897033d6/Configuration/Geometry/scripts/generate2021Geometry.py#L6
* remove deprecated subdets from README

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ianna) [reply Configuration/Geometry/README.md:26]: @kpedro88 - 2021 is still work in progress. There is no corresponding DB payload and it has not been used in production. IMHO, it’s ok to modify it at this point.

File: Configuration/Geometry/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ianna) [reply Configuration/Geometry/README.md:21]: @bsunanda - should we just name them O1 and O2? The scenario has not been used in production yet.

[the remark this answers is not in the document]
Comment (bsunanda) [reply Configuration/Geometry/README.md:21]: I would prefer to keep O1, O2 just as a reference. Here we changed the choice of xml files. Future fixes (if any) will be fixing the xml files

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**